doc/rtd/topics/faq.rst

```diff
@@ -212,17 +242,15 @@ launch this multiple times:
 The above examples all show how to pass user data. To pass other types of
 configuration data use the config option specified below:
 
-+----------------+---------------------+
-| Data           | Config Option       |
-+================+=====================+
-| user data      | user.user-data      |
-+----------------+---------------------+
-| vendor data    | user.vendor-data    |
-+----------------+---------------------+
-| metadata       | user.meta-data      |
-+----------------+---------------------+
-| network config | user.network-config |
-+----------------+---------------------+
++----------------+---------------------------+
+| Data           | Config Option             |
++================+===========================+
+| user data      | cloud-init.user-data      |
```

Comment:
Now that I think of it, we might want to do a broader search of `user.user-data` and replace with `cloud-init.user-data`. I see one a few lines above this change and I know I had it in my tutorial too. Doesn't have to be this PR, but we should add a ticket for it if not.

Reply:
Agreed. I just added a ticket to the backlog for this.